main_cron.py

```diff
@@ -46,6 +46,9 @@
         cron.CronActivitySearchRankHandler),
     main.get_redirect_route(
         r'/cron/jobs/cleanup', cron.CronMapreduceCleanupHandler),
+    main.get_redirect_route(
+        r'/cron/mail/admins/contributor_dashboard_bottlenecks',
```

Comment:
Possible alternative (see longer comment above): contributor_dashboard_reviewer_shortages

Reply:
I like bottlenecks because it's more general (there are two things I'm notifying the admin of here - lack of reviewers and suggestions waiting too long)